docs/sources/awssqs.md

```diff
@@ -1,113 +1,93 @@
 # AWS SQS event source for Knative Eventing
 
-This event source consumes messages from a AWS SQS queue and sends them as CloudEvents to an arbitrary event sink.
-
-## Contents
-
-- [AWS SQS event source for Knative Eventing](#aws-sqs-event-source-for-knative-eventing)
-  - [Contents](#contents)
-  - [Prerequisites](#prerequisites)
-  - [Deployment to Kubernetes](#deployment-to-kubernetes)
-    - [As a AWSSQSSource object](#as-a-awssqssource-object)
-    - [As a ContainerSource object](#as-a-containersource-object)
-    - [As a Deployment object bound by a SinkBinding](#as-a-deployment-object-bound-by-a-sinkbinding)
-  - [Running locally](#running-locally)
-    - [In the shell](#in-the-shell)
-    - [In a Docker container](#in-a-docker-container)
+The event source captures messages sent to a [AWS SQS queue](sqs-docs) and sends them as CloudEvents to an event sink.
 
 ## Prerequisites
 
-* Register an AWS account
-* Create an [Access Key][doc-accesskey] in your AWS IAM dashboard.
-* Create a [SQS queue][doc-sqs].
+### SQS queue
 
-## Deployment to Kubernetes
+If you don't already have an AWS SQS queue, create one by following the instructions in the [Getting started with Amazon SQS][sqs-getting-started] guide.
 
-The _AWS SQS event source_ can be deployed to Kubernetes in different manners:
+### Amazon Resource Name (ARN)
 
-* As an `AWSSQSSource` object, to a cluster where the TriggerMesh _AWS Sources Controller_ is running.
-* As a Knative `ContainerSource`, to any cluster running Knative Eventing.
+A fully qualified ARN is required to uniquely identify the AWS SQS queue.
 
-> :information_source: The sample manifests below reference AWS credentials (Access Key) from a Kubernetes Secret object
-> called `awscreds`. This Secret can be generated with the following command:
->
-> ```console
-> $ kubectl -n <my_namespace> create secret generic awscreds \
->   --from-literal=aws_access_key_id=<my_key_id> \
->   --from-literal=aws_secret_access_key=<my_secret_key>
-> ```
->
-> Alternatively, credentials can be used as literal strings instead of references to Kubernetes Secrets by replacing
-> `valueFrom` attributes with `value` inside API objects' manifests.
+![SQS queue](../images/awssqs-source/sqs-queue.png)
 
-### As a AWSSQSSource object
-
-Copy the sample manifest from `config/samples/awssqssource.yaml` and replace the pre-filled `spec` attributes with the
-values corresponding to your _AWS SQS_ queue. Then, create that `AWSSQSSource` object in your Kubernetes cluster:
+As shown in the above screenshot, you can obtain the ARN of a SQS queue from the AWS console. Alternatively you can also use the [AWS CLI][aws-cli]. The following command retrieves the ARN of a SQS queue named `MyQueue` in the `us-west-2` region.
```

Comment:
c.f. previous comment at line 32 (I just reused the phrasing from https://docs.triggermesh.io/sources/awsdynamodb/ for consistency)

````suggestion
As shown in the above screenshot, you can obtain the ARN of a SQS queue from the AWS console. It typically has the following format:

```
arn:aws:sqs:{awsRegion}:{awsAccountId}:{queueName}
```

Alternatively you can also use the [AWS CLI][aws-cli]. The following command retrieves the ARN of a SQS queue named `MyQueue` in the `us-west-2` region.
````